domains/ciaobot.json

```
@@ -5,6 +5,9 @@
     "discord": "687333016921440317"
   },
   "record": {
-    "A": ["144.91.115.195"]
+    "A": ["144.91.115.195"],
+    "MX": "mx1.improvmx.com.",
+    "MX": "mx2.improvmx.com.",
```

e menomale che programmi in js, dovresti sapere che in un object non puoi avere due key identiche 🤡 
```suggestion
    "MX": [
    	"mx1.improvmx.com",
    	"mx2.improvmx.com"
    ],
```